apps/remix/app/components/general/document-signing/document-signing-text-field.tsx

```diff
@@ -62,7 +62,7 @@ export const DocumentSigningTextField = ({
   const [errors, setErrors] = useState(initialErrors);
   const userInputHasErrors = Object.values(errors).some((error) => error.length > 0);
 
-  const { executeActionAuthProcedure } = useRequiredDocumentSigningAuthContext();
+  const { executeActionAuthProcedure, isEnterprise } = useRequiredDocumentSigningAuthContext();
```

Comment:
Where is `isEnterprise` being used here?

Reply:
Same for the other fields

Reply:
It's not being used, i'll remove it